docs/guide/the-user-interface.md

```diff
@@ -76,12 +76,9 @@ Checking the video caption option will display the acceleration value in the top
 
 %end
 
-- The **Export HTML5 Scene...** item allows you to export the current world as an interactive 3D ".html" file.
-You can get more information about this topic in [this section](web-scene.md).
-
-- ![](images/animation-button.png =26x26) The **Make HTML5 Animation...** item allows you to record a simulation as a 3D animation and publish it on a HTML5 web page.
-Once the animation recording is started, this item is changed to `Stop HTML5 Animation...` and can be used to stop the animation recording.
-You can get more information about this topic in [this section](web-animation.md).
+- ![](images/share-button.png =26x26) The **Share...** item allows you to export the current world as an interactive 3D scene (more information about this topic in [this section](web-scene.md)), or to record a simulation as a 3D animation and publish it on a HTML5 web page.
+Once the animation recording is started, this item is changed to `Stop HTML5 Animation recording.` and can be used to stop the animation recording.
+You can get more information about animations in [this section](web-animation.md).
```

Comment:
We should also describe the two different possibilities (save locally and upload to webots.cloud).
Moreover we mention webots.cloud, but we should also describe somewhere what is webots.cloud.

Reply:
I added a sentence about saving locally or uploading to webots.cloud.

As you suggested, I think it would be a could idea to make some documentation about webots.cloud, but in another PR, and then we can put a link here pointing to this documentation.